test/performance/odp_l2fwd.c

```diff
@@ -2210,7 +2221,16 @@ int main(int argc, char *argv[])
 
 	bind_workers();
 
-	odp_schedule_config(NULL);
+	odp_schedule_config_init(&sched_config);
+	if (odp_schedule_capability(&sched_capa)) {
+		ODPH_ERR("Error: schedule capability failed\n");
+		exit(EXIT_FAILURE);
+	}
+
+	if (gbl_args->appl.flow_aware)
+		sched_config.max_flow_id = sched_capa.max_flow_id;
```

Comment:
Perhaps it would be good to check here if flow aware mode is supported and throw an error + exit if not. Currently, a user has no way of knowing if the platform actually supports flow aware mode.

Reply:
I will add it in the next version.